stable/fairwinds-insights/templates/rbac-repo-scan-job.yaml

```diff
@@ -25,6 +25,9 @@ rules:
 - apiGroups: [""]
   resources: ["events"]
   verbs: ["list", "get"]
+- apiGroups: [""]
+  resources: ["secrets"]
+  verbs: ["create", "delete", "get"]
```

Comment:
do we need `get`? That one is kinda scary

Reply:
I don't think we need it - I just checked... we create and delete based on known secret name so...

Reply:
Done, removed `get` from the verbs